app/config.py

```diff
@@ -251,7 +251,7 @@ class Config(object):
             },
             "delete_old_s3_objects": {
                 "task": "delete-old-s3-objects",
-                "schedule": crontab(minute="*/5"),
+                "schedule": crontab(hour=7, minute=10),
```

Comment:
This runs the task less often correct? Just wanted to understand what this change was for.

Reply:
Yes, run the task once a day instead of every 5 minutes.  Every 5 minutes was just for debugging.